### PR TITLE
used different schema for zkPass and Reclaim protocol

### PR DESCRIPTION
**Pull request by icetrust0212**

### CODE REVIEW

File: hooks/utils.ts [hunk 1]
```diff
@@ -1,36 +1,50 @@
...
+  schema: Schema
+) => {
   // Note: `context` should already be obtained by connecting to the Verida Network
   const credentialSDK = new Credentials();
 
   // The Verida DID that is the subject of this credential (who is being verified with this credential?)
   // const veridaDid = 'did:vda:testnet:0x7c9699514D3e150e6F50d63345D4bD7B9831da7f';
 
-  const credentialSchema =
+  const zkPassCredentialSchema =
```
Comment (tahpot) [inline hooks/utils.ts]: Maybe put this in the `providers` config file, using an array keyed by `src`?

File: pages/api/send-message.ts [hunk 2]
```diff
@@ -17,7 +16,7 @@ export default async function handler(
       process.env.IS_DEV === "true"
         ? EnvironmentType.TESTNET
         : EnvironmentType.MAINNET;
-    const CONTEXT_NAME = "Dapp Connector";
+    const CONTEXT_NAME = "Proof Connector";
```
Comment (tahpot) [inline pages/api/send-message.ts]: Should be `Verida: Proof Connector`

File: pages/api/send-message.ts [hunk 3]
```diff
@@ -64,7 +62,10 @@ export default async function handler(
       const messaging = await context?.getMessaging();
       const type = "inbox/type/dataSend";
 
-      const message = `zkPass credential: ${schema.host}`;
+      const message =
+        schema.src === "zkPass"
```
Comment (tahpot) [inline pages/api/send-message.ts]: `New Credential Proof: ${schema.host}`

**Review by tahpot — CHANGES_REQUESTED** [review]

File: hooks/utils.ts (hunk 1 above)
Comment (icetrust0212) [reply hooks/utils.ts]: Done

File: pages/api/send-message.ts (hunk 2 above)
Comment (icetrust0212) [reply pages/api/send-message.ts]: Done

File: pages/api/send-message.ts (hunk 3 above)
Comment (icetrust0212) [reply pages/api/send-message.ts]: Done

**Review by tahpot — APPROVED** [review]

**Pull request merged**